packages/@uppy/robodog/src/addTransloaditPlugin.js

```diff
@@ -3,6 +3,7 @@ const has = require('@uppy/utils/lib/hasProperty')
 const TransloaditResults = require('./TransloaditResultsPlugin')
 
 const transloaditOptionNames = [
+  'id',
   'service',
```

Comment:
```suggestion
  'service',
```
`id` is intentionally\* not passed in because there isn't really a reason to rename plugin instances with Robodog

Reply:
I think it makes sense. 

My only suggestion then is that I would also need to modify these docs so they don't say "All the options to the Transloadit plugin are supported." https://uppy.io/docs/robodog/dashboard/#Transloadit 

"Most options to the Transloadit plugin are supported" ?